wasm2c/wasm-rt-exceptions.h

```diff
@@ -17,6 +17,10 @@
 #ifndef WASM_RT_EXCEPTIONS_H_
 #define WASM_RT_EXCEPTIONS_H_
 
+#ifdef __cplusplus
```

Comment:
Maybe this should go after the `#include "wasm-rt.h"` ?

Reply:
fixed